retrieval_service/datastore/providers/postgres.py

```diff
@@ -328,5 +348,62 @@ async def search_flights_by_airports(
         results = [models.Flight.model_validate(dict(r)) for r in results]
         return results
 
+    async def insert_ticket(
+        self,
+        user_id: int,
+        user_name: str,
+        user_email: str,
+        airline: str,
+        flight_number: str,
+        departure_airport: str,
+        arrival_airport: str,
+        departure_time: datetime.datetime,
+        arrival_time: datetime.datetime,
+    ) -> list[models.Ticket]:
+        result = await self.__pool.execute(
+            """
+                INSERT INTO tickets (
```

Comment:
Should we validate that the flight exists?

Reply:
added validation